applications/DEMApplication/custom_problemtype/DEMpack/F-DEMPack/kratos.gid/kratos.unix.bat

```diff
@@ -32,5 +32,5 @@ export LD_LIBRARY_PATH="$3/kratos":"$3/kratos/libs"
 export OMP_NUM_THREADS=$5
 
 # Run Python using the script KratosSwimmingDEM.py
-#"$3/kratos/runkratos" KratosSwimmingDEM.py > "$2/$1.info" 2> "$2/$1.err"
-python3 KratosSwimmingDEM.py > "$2/$1.info" 2> "$2/$1.err"
+#"$3/kratos/runkratos" KratosSwimmingDEMAnalysis.py > "$2/$1.info" 2> "$2/$1.err"
+python3 KratosSwimmingDEMAnalysis.py > "$2/$1.info" 2> "$2/$1.err"
```

Comment:
We should leave runkratos for the moment. It's better in case we pack some release.